### ISSUE TITLE
ERROR : nodeBooter - Unable to launch DeviceManager softpkg: No matching implementation found.

### ISSUE THREAD

**Issue by NayanaAnand** [Geontech/meta-redhawk-sdr#61]
Hi,

I was trying to include meta-redhawk-sdr layer(Branch : krogoth) to Yocto build and build was successful.

After loading the image to hardware, found OmniNames and OminEvents task on brackground, after running the script "node-DevMgr-<MACHINE_NMAE>-ALL" manullay.

But Unable to launch Device/Domain manager with daemon "nodeBooter -d/D <*.dcd.xml>". Observed below error:
ERROR : nodeBooter - Unable to launch DeviceManager softpkg: No matching implementation found.
 
Can anyone suggest us is any packages needs to install/include during the build.

Regards,
Nayana

**Comment by NayanaAnand** [comment]
Issue is resolved after changing branch to pyro.

**Comment by btgoodwin** [comment]
I apologize for my tardiness in reaching out to you.  At the start of the year, I begin regression testing more recent versions of REDHAWK and Yocto on a series of -next branches.  There are a raft of new features in there, some related to device manager generation and creation of init.d/systemd service units.

Some downsides to these new builds is I'm seeing odd behaviors in how the GPP scrapes /proc with regular expressions, for example, and how the ComponentHost will fail to load the `rh.DataConverter` component because of it depending on a SoftPkg that depends on another SoftPkg (they load out of order, causing one to complain about the other being missing even though it is installed and ready to be loaded _next_).  These issues were all discovered and I folded in OE QA automated testing to help me more rapidly regress Yocto versions and machine types to figure out exactly why aarch64 shows some but not armv7l, and why x86_64 is also showing some even going back to Krogoth.

Unfortunately I have not had time to continue sorting through those issues.  I'm going to take a look at your others now as it has been a very long time since I've looked at Pyro.  I'm glad that got you past this, however.
